src/msg/async/Event.h

```diff
@@ -113,7 +113,7 @@ class EventCenter {
   CephContext *cct;
   int nevent;
   // Used only to external event
-  pthread_t owner;
+  pthread_t owner = 0;
```

Comment:
> <what>Conditional jump or move depends on uninitialised value(s)</what>

the EventCenter should have been bound with its worker thread at that moment, right?